src/org/zaproxy/zap/extension/autoupdate/AddOnsTableModel.java

```diff
@@ -53,7 +53,7 @@
 
     protected final Logger logger = Logger.getLogger(this.getClass());
 
-    private final Comparator<AddOnWrapper> comparator;
+    private Comparator<AddOnWrapper> comparator;
```

Comment:
This can be kept final, you just need to initialise it to null in the new constructor.